src/components/Profile/ProfileComponent.jsx

```diff
@@ -58,7 +71,11 @@ const ProfileComponent = ({ clientProfile }) => {
       nickname
```

Comment:
same here `inputValuesPublic` and `inputValuesPrivate`

Reply:
I believe the existing lines work to distinguish the two?